client/account-recovery/lost-password/lost-password-form/index.jsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import support from 'lib/url/support';
+import Card from 'components/card';
+import Button from 'components/button';
+import FormLabel from 'components/forms/form-label';
+import FormInput from 'components/forms/form-text-input';
+
+export class LostPasswordFormComponent extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.state = {
+			isSubmitting: false,
+			userLogin: '',
+		};
+	}
+
+	onSubmit = () => {
+		this.setState( { isSubmitting: true } );
+
+		//TODO: dispatch an event with userLogin and wait to here back
+	};
```

Comment:
Renamed to `submitForm` in https://github.com/Automattic/wp-calypso/pull/8955/commits/18084a9a7cd099fc4afe3941b02e5b044adbb677

I prefer to keep the curly braces since I have a comment in there and there will be some more logic in there in my next pr.